Build errors

## Before running cargo update
```
error[E0635]: unknown feature `stdsimd`
  --> /home/yonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ahash-0.8.6/src/lib.rs:99:42
   |
99 | #![cfg_attr(feature = "stdsimd", feature(stdsimd))]
```

## After running cargo update
```
$ cargo build --release
   Compiling rss-funnel v0.1.0 (/git/rss-funnel)
error: #[derive(RustEmbed)] folder '/git/rss-funnel/inspector/dist/' does not exist. cwd: '/git/rss-funnel'
  --> src/server/inspector.rs:13:1
   |
13 | / #[folder = "inspector/dist/"]
14 | | struct Asset;
   | |_____________^

error[E0599]: no function or associated item named `get` found for struct `Asset` in the current scope
  --> src/server/inspector.rs:47:24
   |
14 | struct Asset;
   | ------------ function or associated item `get` not found for this struct
...
47 |   let content = Asset::get(path.as_str()).map(|content| content.data);
   |                        ^^^ function or associated item not found in `Asset`
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following traits define an item `get`, perhaps you need to implement one of them:
           candidate #1: `SliceIndex`
           candidate #2: `rustls::server::server_conn::StoresServerSessions`
           candidate #3: `string_cache::static_sets::StaticAtomSet`
           candidate #4: `RustEmbed`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rss-funnel` (bin "rss-funnel") due to 2 previous errors
```

OS: FreeBSD 14 and Ubuntu 23
Rust: `rustc 1.78.0-nightly (6672c16af 2024-02-17)`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,6 +11,10 @@ documentation = "https://docs.rs/rss-funnel"
 keywords = ["rss", "feed", "cli"]
 categories = ["command-line-interface", "web-programming"]
 
+[features]
+default = ["inspector-ui"]
+inspector-ui = ["dep:rust-embed"]
+
 [dependencies]
 # Async runtime, utility and helper crates
 async-trait = "0.1.74"
@@ -42,7 +46,7 @@ mime = "0.3.17"
 
 # webui (inspector) support
 maud = { version = "0.26.0", features = ["axum"] }
-rust-embed = { version = "8.2.0", features = ["include-exclude"] }
+rust-embed = { version = "8.2.0", features = ["include-exclude"], optional = true }
 
 # HTML manipulation in the feeds
 scraper = "0.18.1"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,7 +45,6 @@ url = { version = "2.5.0", features = ["serde"] }
 mime = "0.3.17"
 
 # webui (inspector) support
-maud = { version = "0.26.0", features = ["axum"] }
 rust-embed = { version = "8.2.0", features = ["include-exclude"], optional = true }
 
 # HTML manipulation in the feeds

--- a/src/config.rs
+++ b/src/config.rs
@@ -1,3 +1,4 @@
+#[cfg(feature = "inspector")]
 pub use crate::cli::FeedDefinition;
 // pub use crate::client::ClientConfig;
 // pub use crate::filter::FilterConfig;

--- a/src/server.rs
+++ b/src/server.rs
@@ -1,4 +1,5 @@
 pub(crate) mod endpoint;
+#[cfg(feature = "inspector")]
 mod inspector;
 
 use std::{path::Path, sync::Arc, time::Duration};
@@ -24,7 +25,7 @@ pub struct ServerConfig {
   bind: Arc<str>,
 
   /// Whether to enable the inspector UI
-  #[clap(
+  #[cfg_attr(clap,
     long,
     action = clap::ArgAction::Set,
     num_args = 0..=1,
@@ -32,6 +33,7 @@ pub struct ServerConfig {
     default_value = "true",
     default_missing_value = "true"
   )]
+  #[cfg(feature = "inspector")]
   inspector_ui: bool,
 
   /// Watch the config file for changes and restart the server
@@ -104,9 +106,14 @@ impl ServerConfig {
       app = app.merge(endpoint_route);
     }
 
+    #[cfg(feature = "inspector")]
     if self.inspector_ui {
       app = app.nest("/", inspector::router(feed_definition))
     } else {
+      app = app.route("/", get(|| async { "rss-funnel is up and running!" }));
+    }
+
+    if !cfg!(feature = "inspector") {
       app = app.route("/", get(|| async { "rss-funnel is up and running!" }));
     }
 

--- a/src/server/inspector.rs
+++ b/src/server/inspector.rs
@@ -4,7 +4,6 @@ use axum::response::{IntoResponse, Redirect, Response};
 use axum::Json;
 use axum::{routing::get, Extension, Router};
 use http::{StatusCode, Uri};
-// use maud::{html, Markup};
 
 use crate::config::{self, FeedDefinition};
 use crate::util::Error;

